src/Handler/MockHandler.php

```diff
@@ -83,8 +84,10 @@ public function __invoke(RequestInterface $request, array $options): PromiseInte
             throw new \OutOfBoundsException('Mock queue is empty');
         }
 
-        if (isset($options['delay']) && \is_numeric($options['delay'])) {
-            \usleep($options['delay'] * 1000);
+        /** @var ?int $delay */
+        $delay = $options['delay'] ?? null;
+        if (\is_numeric($delay)) {
+            \usleep($delay * 1000);
```

Comment:
well again here.. I prefer having a baseline error rather just have some phpstan optimization for the sake of it...

Reply:
Yeh, this is unnecessary.

Reply:
reverted in e05fc9539d28e1b7d145079c1e7a7e572fb3e94e